FWCore/Framework/src/EventSetupsController.cc

```diff
@@ -354,6 +354,23 @@ namespace edm {
       return nullptr;
     }
 
+    ParameterSet* EventSetupsController::getESProducerPSet(ParameterSetID const& psetID, unsigned subProcessIndex) {
```

Comment:
This function is currently just a copy-paste of the `const` member function above. If we accept this approach, I'd look to avoid the code duplication.